src/utils/dayViewLayout.js

```diff
@@ -355,10 +358,56 @@ export function getStyledAvailabilities ({
     totalMin,
   };
 
-  availabilities.forEach((availability, availabilityIdx) => {
-    let style = getYStyles(availabilityIdx, helperArgs);
-    styledAvailabilities.push({ availability, style });
-  })
+  const overlapMap = {};
+  let columnIndex = 0;
+
+  for (let availabilityIndex = 0; availabilityIndex < availabilities.length; availabilityIndex++) {
```

Comment:
Is this reusing the existing overlap logic for events at all? I would have expected a refactor to a shared function that can be used to handle overlaps for both events and availabilities. Or are availability overlaps special in some way?

Reply:
Availability overlaps don't stack like events. They live side-by-side. But refactoring styled events/availabilities into a single shared function will likely happen in the near future.

Reply:
I see, thanks for the explanation.